lib/icedfrisby.js

```diff
@@ -286,14 +283,13 @@ class Frisby {
 
 
   /**
-   * @param {object} headers - header object {k;v, k:v}
+   * @param {object} headers - header object {k:v, k:v}
    * @return {object}
    * @desc Add group of HTTP headers together
    */
   addHeaders (headers) {
-    const self = this
-    _.forEach(headers, function(val, key) {
-      self.addHeader(key, val)
+    Object.keys(headers).forEach(key => {
```

Comment:
Just food for thought, we could also use lodash's `forEach` here:

```js
_.forEach({ 'a': 1, 'b': 2 }, function(value, key) {
  console.log(value, key);
});
```

Reply:
Yea. I'm ambivalent about this one. I'm inclined to use lodash where it's clearer or less error prone than the ES6 version. I mean, we shouldn't implement our own `pluck` or `merge` function. In this case I feel like it's equally clear either way. It's not error prone.